api4/team.go

```diff
@@ -385,7 +387,13 @@ func getTeamMembers(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	members, err := c.App.GetTeamMembers(c.Params.TeamId, c.Params.Page*c.Params.PerPage, c.Params.PerPage, restrictions)
+	excludeDeletedUsersBool, _ := strconv.ParseBool(excludeDeletedUsers)
```

Comment:
Can we move this up to be near where we access the query string? It's part of the `Parsing of request URL and body` step which should come first (as per https://developers.mattermost.com/contribute/server/rest-api/).

Reply:
Thanks for pointing this out. Will keep in mind for future. Done!